universal-application-tool-0.0.1/test/FunctionalTest.java

```diff
@@ -70,6 +70,17 @@ public void addPerson() {
     assertThat(contentAsString(result)).contains("person John with ID:", "added.");
   }
 
+  @Test
+  public void syncAddPerson() {
+    Http.RequestBuilder request =
+        fakeRequest(routes.PostgresController.syncAdd("John"))
+            .header(Http.HeaderNames.HOST, "localhost:" + testServerPort());
+    Result result = route(app, request);
+
+    assertThat(result.status()).isEqualTo(OK);
+    assertThat(contentAsString(result)).contains("person John with ID:", "synchronously added.");
+  }
+
```

Comment:
Move this test case and the other related to `PersonController` into a file and class specific to `PersonalController`. A directory structure would be good here:
```
test/functional/PersonControllerTest.java
```